server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -124,7 +116,7 @@
  */
 public class StorageGroupProcessor {
 
-  private static final String MERGING_MODIFICATION_FILE_NAME = "merge.mods";
+  public static final String MERGING_MODIFICATION_FILE_NAME = "merge.mods";
```

Comment:
it would be better to set it as a private variable

Reply:
TsFileManage also use it